src/tests/Parser/Brewmaster/SimpleFIght.js

```diff
@@ -0,0 +1,88 @@
+import SPELLS from 'common/SPELLS';
+
+export const FIGHT_END = 11000;
+export const TOTAL_STAGGERED = 231;
+
+const thisPlayer = 1;
+const enemy = 2;
+
+export function processEvents(events, module, fightEndTime) {
+  if (!fightEndTime) {
+    fightEndTime = FIGHT_END;
+  }
+  events.forEach(event => {
+    if (event.timestamp <= fightEndTime) {
```

Comment:
You should use fireEvent instead; it's also being tested iirc so it should always work properly and make for cleaner test code.